src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -553,34 +553,39 @@ public void StopStake()
                 .GetSpendableTransactionsInWallet(walletSecret.WalletName, 1).ToList();
 
             FetchCoinsResponse fetchedCoinSet = await this.coinView.FetchCoinsAsync(spendableTransactions.Select(t => t.Transaction.Id).ToArray(), cancellationToken).ConfigureAwait(false);
-            if (cancellationToken.IsCancellationRequested) return utxoStakeDescriptions;
+            if (cancellationToken.IsCancellationRequested)
+                return utxoStakeDescriptions;
 
             foreach (UnspentOutputReference outputReference in spendableTransactions)
             {
                 UnspentOutputs coinSet = fetchedCoinSet.UnspentOutputs
                     .FirstOrDefault(f => f?.TransactionId == outputReference.Transaction.Id);
-                if (coinSet == null || outputReference.Transaction.Index >= coinSet.Outputs.Length) continue;
+                if (coinSet == null || outputReference.Transaction.Index >= coinSet.Outputs.Length)
```

Comment:
Brackets here and below as mentioned earlier

Reply:
Please add the braces